spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -128,6 +127,8 @@
  * @author Allard Buijze
  * @since 3.0
  */
+
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.